app/views/incomeSources/add/ForeignPropertyCheckDetails.scala.html

```
@@ -0,0 +1,123 @@
+        govukTable: GovukTable,
+        govukButton: GovukButton,
+        formWithCSRF: FormWithCSRF
+)
+
+@(
+        viewModel: CheckForeignPropertyViewModel,
+        postAction: Call,
+        isAgent: Boolean,
+        backUrl: String,
+        origin: Option[String] = None
+)(implicit messages: Messages, user: auth.MtdItUser[_])
+
+@import implicitDateFormatter.longDate
+
+@getMessage(key: String, args: String*) = @{
+    messages(s"incomeSources.add.foreign-property-check-details.$key", args: _*)
+}
+
+@changeForeignPropertyStartDateURL = @{
+    if(isAgent) {
+        controllers.incomeSources.add.routes.ForeignPropertyStartDateController.changeForeignPropertyStartDateAgent().url
+    } else {
+        controllers.incomeSources.add.routes.ForeignPropertyStartDateController.changeForeignPropertyStartDate().url
+    }
+}
+
+@changeForeignPropertyAccountingMethodURL = @{
+    if(isAgent) {
+        controllers.incomeSources.add.routes.ForeignPropertyAccountingMethodController.changeForeignPropertyAccountingMethodAgent().url
+    } else {
+        controllers.incomeSources.add.routes.ForeignPropertyAccountingMethodController.changeForeignPropertyAccountingMethod().url
+    }
+}
+
+@mainTemplate(
+    pageTitle = getMessage("title"),
+    backUrl = Some(backUrl),
+    isAgent = isAgent,
+    btaNavPartial = user.btaNavPartial) {
+
+    @hmrcPageHeading(PageHeading(
+        text = getMessage("title"),
+        headingClasses = Some("govuk-heading-l"),
+        section = Some(getMessage("heading")),
+        captionClasses = Some("govuk-caption-l")
+    ))
+
+    @formWithCSRF(action = postAction) {
+
+        <dl class="govuk-summary-list" id="refund-to-taxpayer-table">
+            <div class="govuk-summary-list__row">
+                <dt class="govuk-summary-list__key">@getMessage("start-date")</dt>
+                <dd class="govuk-summary-list__value" id="foreign-property-date-value">@{
+                    viewModel.tradingStartDate.get.toLongDate
```

tradingStartDate - this field should really start as mandatory in the view model so we are not calling .get over Option; when mapping we can raise error if this date is not provided;